src/continuous_agg.c

```diff
@@ -50,6 +50,11 @@ static const WithClauseDefinition continuous_aggregate_with_clause_def[] = {
 			.arg_name = "max_interval_per_job",
 			.type_id = TEXTOID,
 		},
+		[ContinuousViewOptionCreateGroupIndex] = {
+			.arg_name = "create_groupcol_index",
```

Comment:
I would name this `create_group_indexes` or something like that.

Reply:
I do like `create_group_indexes`

Reply:
modified.